src/components/viewer/ViewerItem.tsx

```diff
@@ -206,7 +208,7 @@ export default class ViewerItem extends React.Component<Partial<IViewerItemProps
             <Popup
               trigger={this.upvoteButton()}
               size="small"
-              className="pp-ui pp-popup-small-padding"
+              className={classNames(indirectChildClassName, 'pp-ui', 'pp-popup-small-padding')}
```

Comment:
can we extract these two classes to some constants file?

Reply:
I'd love to. I didn't want to mix it with other changes (I guess `pp-ui` appears in some 15 places now). It's coming in the next PR ;)